### DISCUSSION TITLE
Edit Domain resets when reopening orange

### DISCUSSION THREAD

**Discussion by zikook** [biolab/orange3#6412] [Q&A]
Hi,
When I close and reopen Orange, changes I've made in Edit Domain, for example naming a variable, sometimes disappear.
This happens when I use the widget 'Create Table' from the Educational tab to send it a table.
I can upload a workflow if it helps.
Anyone experienced this? Is there a workaround?
Thanks,
-Aaron

**Comment by janezd** [comment]
@zikook, thanks. You can upload the workflow (please do), but I'm pretty sure what I'll find there. And it's a bit complicated to explain.

Note to self (or anybody): this widget shouldn't have context settings but a rather permanent-ish storage of "variable hints".

"Permanent-ish": not storing more than 1000 variables or something, so they're not kept indefinitely. LRU.

**Comment by zikook** [comment]
Hi,
Thanks for replying.
I've attached the workflow, zipped because github doesn't accept .ows files. 
Best,
-Aaron
[example.ows.zip](https://github.com/biolab/orange3/files/11240321/example.ows.zip)

**Comment by janezd** [comment]
Thanks. It's as I expected. I can't be absolutely sure what's happening, but if I'm correct, #6415 should fix this.

**Reply by zikook** [reply]
Thanks!
